micromamba/src/env.cpp

```diff
@@ -284,6 +284,14 @@ set_env_command(CLI::App* com, Configuration& config)
 
     update_subcom->callback(
         [&config]
-        { update(config, /*update_all*/ false, /*prune_deps*/ false, remove_not_specified); }
+        {
+            update(
+                config,
+                /*update_all*/ false,
```

Comment:
It would be worth, in addition to have a struct with these options, to use `enum class` values to set them so that it's clear just by reading, without comments (which could go out of sync). C++20 also allows to `{ .update_all = false, ... }` which would be useful too.